integrations/access/servicenow/app.go

```diff
@@ -316,7 +357,7 @@ func (a *App) onDeletedRequest(ctx context.Context, reqID string) error {
 	return a.resolveIncident(ctx, reqID, Resolution{State: ResolutionStateResolved})
 }
 
-func (a *App) getOnCallServiceNames(req types.AccessRequest) ([]string, error) {
+func (a *App) getOnCallServiceNames(ctx context.Context, req types.AccessRequest) ([]string, error) {
```

Comment:
Why did we add a `context.Context` param here? I'm fine with it, mostly curious because it doesn't seem to be used.

Reply:
Was debugging something and forgot to remove it again. It is not needed, will remove it now.